style(src): improve clause components a11y - I57

elit-altum opened the pull request:
# Issue #57
I have checked and modified all the elements which weren't complying with the WCAG 2.0 Standards i.e. the contrast ratio was lesser than 4.5:1

### Changes
- The following graphic should explain the changes made in a descriptive manner

![Color-Codes-Solved](https://user-images.githubusercontent.com/41413622/75228256-d74cb300-57d5-11ea-9ef2-8d38649a2c50.png)

### Related Issues
- Builds upon the Pull Request #202

irmerk commented:
Could you update your commit and PR title with [this as a guide](https://github.com/accordproject/cicero-ui/blob/master/DEVELOPERS.md#-git-commit-guidelines)?

elit-altum commented:
> 
> 
> Could you update your commit and PR title with [this as a guide](https://github.com/accordproject/cicero-ui/blob/master/DEVELOPERS.md#-git-commit-guidelines)?

Really sorry for this @irmerk 
Would this be an appropriate commit message :
```
style: fixes clause components accessibility to WCAG2.0 - Fixes #57
```

irmerk commented:
I think this works: `style(src): improve clause components a11y - I57`

elit-altum commented:
> 
> 
> I think this works: `style(src): improve clause components a11y - I57`

Thank you for your help @irmerk . 
I have squashed the commits and updated my branch as well. Please review!

elit-altum commented:
@Michael-Grover Sorry but I don't know how the review was dismissed. I think it was dismissed due to a force push I made after updating my branch. As mentioned in https://github.com/isaacs/github/issues/1157. I did not do it intentionally, I am sorry for any inconvenience caused.
